client/sites/staging-site/components/staging-site-sync-modal/index.tsx

```diff
@@ -27,10 +27,20 @@ import { usePullFromStagingMutation } from 'calypso/sites/staging-site/hooks/use
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import getBackupBrowserCheckList from 'calypso/state/rewind/selectors/get-backup-browser-check-list';
 import { getSiteSlug } from 'calypso/state/sites/selectors';
+import type { FileBrowserFilterConfig } from 'calypso/my-sites/backup/backup-contents-page/file-browser';
 
 // TODO: Temporary style for the PoC
 import './style.scss';
 
+const filterConfig: FileBrowserFilterConfig = {
```

Comment:
I wonder if we don't want to keep `sql` as well - since in Studio we have a dedicated checkbox for database:

![Markup on 2025-07-09 at 14:06:59](https://github.com/user-attachments/assets/d74784e5-046a-4741-81dd-337d064aeb77)

Reply:
Good point! I think we can handle that as part of a follow-up task so we can test it independently. I wll create a task for that 👍